plugins/analytics/public/process_ip.py

```diff
@@ -19,18 +18,14 @@
     reader = False
 
 
-
-
-class ProcessIp(ScheduledAnalytics):
+class ProcessIp(InlineAnalytics):
```

Comment:
Arguably IPs tend to change very little over time, but is different Geo location not something we'd like to track over time?